### PR TITLE
style: Fixed UI of Privacy Fragment

### LINKED ISSUE #2414: UI of Privacy fragment
**Scope of improvement in UI of privacy fragment**

Cards of privacy fragment are not very pleasing, also there is scope of improvements in padding, Margin, font etc

Screenshots
untitled4

I've worked on this issue
**Pardon for the first contribution, I've created PR #2412  to this issue before filing it**

### PR DESCRIPTION
**Pull request by pallav12**
Fixes: #2414 

Changes: Removed the cards, Fixed margins and paddings, Changed font style

Screenshots for the change: 
untitled
untitled1

### CODE REVIEW

**Comment by atm1504** [comment]
@pallav12 please reference an issue with this pr

**Comment by pallav12** [comment]
@atm1504 do I have to address a new issue or target existing issues in my contribution?? 
pardon this is my first PR with this project

**Comment by atm1504** [comment]
if the issue is present then target that issue, else make an issue first and then refer it.

**Comment by pallav12** [comment]
thanks, @atm1504 ... that makes more sense also

File: app/src/main/res/layout/fragment_privacy.xml [hunk 1]
```diff
@@ -15,383 +14,361 @@
...
-            </android.support.v7.widget.CardView>
-
-            <android.support.v7.widget.CardView
+                    android:layout_marginLeft="@dimen/margin_moderate"
+                    android:text="@string/privacy_welcome_details"
+                    android:textColor="@color/md_black_1000"
+                    android:textSize="@dimen/text_size_normal"
+                    android:typeface="monospace" />
+            </LinearLayout>
+
```
Comment (atm1504) [inline app/src/main/res/layout/fragment_privacy.xml:48]: Remove unwanted blank lines

File: app/src/main/res/layout/fragment_privacy.xml [hunk 2]
```diff
@@ -15,383 +14,361 @@
...
+            <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_marginTop="@dimen/margin_moderate"
-                android:paddingLeft="@dimen/padding_moderate"
-                card_view:cardCornerRadius="@dimen/card_radius_privacy">
+                android:orientation="vertical"
+                android:paddingLeft="@dimen/padding_small"
+                android:paddingRight="@dimen/padding_small"
+
```
Comment (atm1504) [inline app/src/main/res/layout/fragment_privacy.xml]: same

File: app/src/main/res/layout/fragment_privacy.xml [hunk 3]
```diff
@@ -15,383 +14,361 @@
...
-
-            <android.support.v7.widget.CardView
+                    android:layout_marginLeft="@dimen/margin_moderate"
+                    android:text="@string/privacy_services_details"
+                    android:textColor="@color/md_black_1000"
+                    android:textSize="@dimen/text_size_normal"
+                    android:typeface="monospace" />
+            </LinearLayout>
+
+
```
Comment (atm1504) [inline app/src/main/res/layout/fragment_privacy.xml]: same

File: app/src/main/res/layout/fragment_privacy.xml [hunk 4]
```diff
@@ -1,10 +1,9 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:card_view="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:layout_marginTop="@dimen/margin_moderate"
     android:layout_marginBottom="@dimen/margin_moderate"
+    android:background="#fff"
```
Comment (atm1504) [inline app/src/main/res/layout/fragment_privacy.xml]: Use background color from colors file. Don't hardcode colors value in the xml file

File: app/src/debug/AndroidManifest.xml [hunk 5]
```diff
@@ -7,7 +7,7 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-
+    <meta-data android:name="io.fabric.ApiKey" android:value="fabric_api_key" />
```
Comment (atm1504) [inline app/src/debug/AndroidManifest.xml]: why?

**Review by atm1504 — CHANGES_REQUESTED** [review]

File: app/src/debug/AndroidManifest.xml (hunk 5 above)
Comment (pallav12) [reply app/src/debug/AndroidManifest.xml]: that was an attempt to resolve build issue... mentioned in readme, but now its redundant. Updated according to review and squashed into single commit.

**Review by atm1504 — APPROVED** [review]

**Pull request merged**